src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyInformationPoint.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Xml;
+using Altinn.Authorization.ABAC.Constants;
+using Altinn.Authorization.ABAC.Utils;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Platform.Authorization.Configuration;
+using Altinn.Platform.Authorization.Helpers;
+using Altinn.Platform.Authorization.Models;
+using Altinn.Platform.Authorization.Repositories;
+using Altinn.Platform.Authorization.Repositories.Interface;
+using Altinn.Platform.Authorization.Services.Interface;
+using Azure;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
```

Comment:
There are multiple unnecessary namespace imports we can remove.

Reply:
Removed them.